client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverImpersonateUser.java

```diff
@@ -47,17 +50,21 @@
 
     @BeforeClass
     public void setup()
+            throws IOException
     {
+        Path passwordConfigDummy = Files.createTempFile("passwordConfigDummy", "");
```

Comment:
Why not use `(null, null)` like `TestPasswordAuthenticatorConfig`? Seems like they should be consistent

Reply:
The prefix is defensive. If you happen to find 5k of files in `/tmp` you know where to look for a bug. Suffix changed to null